prisma/migrations/20220419202419_initial_migration/migration.sql

```diff
@@ -0,0 +1,60 @@
+-- CreateEnum
```

Comment:
"Squashed" the migrations. Given we don't have any release yet, I thought it made sense to have only one migration.